utils/database.js

```diff
@@ -67,30 +67,26 @@ class Database {
   ) {
     let baniLength;
     switch (length) {
-      case "EXTRA_LONG":
-        baniLength = "existsBuddhaDal";
+      case 'EXTRA_LONG':
+        baniLength = 'existsBuddhaDal';
         break;
-      case "LONG":
-        baniLength = "existsTaksal";
+      case 'LONG':
+        baniLength = 'existsTaksal';
         break;
-      case "MEDIUM":
-        baniLength = "existsMedium";
+      case 'MEDIUM':
+        baniLength = 'existsMedium';
         break;
-      case "SHORT":
-        baniLength = "existsSGPC";
+      case 'SHORT':
+        baniLength = 'existsSGPC';
         break;
       default:
-        baniLength = "existsMedium";
+        baniLength = 'existsMedium';
     }
     return new Promise(function (resolve) {
       Database.initDB().then(() => {
         db.executeSql(
-          `SELECT ID, Paragraph, header, Gurmukhi, Visraam, Transliterations, Translations FROM mv_Banis_Shabad WHERE Bani = ${ 
-          baniId 
-          } AND ${ 
-          baniLength 
-          } = 1 AND (MangalPosition IS NULL OR MangalPosition = ${ 
-          mangalPosition === "CURRENT_SAROOPS" ? "'current'" : "'above'" 
+          `SELECT ID, Paragraph, header, Gurmukhi, Visraam, Transliterations, Translations FROM mv_Banis_Shabad WHERE Bani = ${baniId} AND ${baniLength} = 1 AND (MangalPosition IS NULL OR MangalPosition = ${
+            mangalPosition === 'CURRENT_SAROOPS' ? "'current'" : "'above'"
```

Comment:
Veerji, are mangal positions still in use in this application? We might have to confirm this from @prebhdevsingh veerji if we have to remove it from this application or not.

Reply:
I commented manglacharan part from front end. However, the logic and database calling still exists.